Lesson_08/l08_h01_max_three_multiplication.py

```diff
@@ -0,0 +1,26 @@
+"""
+HW:8[Volkov Anton].
+
+Given the list of integers ( positive , negative, zeros)
+Find max multiplication of three values in list
+l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+
+Input: [-1, 1, 2, 0, 3, 12, 4, 5, -1, 6]
+Output: Max value = "360". Nums are: (12, 5, 6)
+
+Input: [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+Output: Max value = "2016". Nums are: (-7, 12, -24)
+"""
+l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+multiplication_numbers = ()
+max_value = 0
+
+for i in range(len(l1) - 2):
```

Comment:
As advanced level try to understand that your algorithm has complexity O(n**3)
But somehow could be found solution with complexity O(n)
Advice: sort list and try to find multiplication maximum possible patterns